src/main/resources/archetype-resources/Jenkinsfile

```
@@ -2,4 +2,4 @@
 
 library('jenkins-libraries@v3.2.1')
```

```suggestion
library('jenkins-libraries@v3.3.0')
```
I just created a new release you can use
https://github.com/rebuy-de/jenkins-libraries/releases/tag/v3.3.0

---

Updated, thanks!